src/pages/ProgramList/index.tsx

```diff
@@ -1,8 +1,45 @@
 import React from "react";
 
+import { useQuery } from "react-query";
+
+import { Alert } from "@mui/material";
+import { styled } from "@mui/system";
+
+import { Program } from "../../shared/interfaces";
+import { ProgramsService } from "../../services";
+
+import { ProgramItem, SectionHeader, SectionContainer } from "../../components";
+
 const ProgramListPage: React.FC = () => {
-  return <div>ProgramList</div>;
+  const {
+    status,
+    error,
+    data: programs,
+  } = useQuery(["programs"], ProgramsService.getAllPrograms);
```

Comment:
enum
do wywołania konkretnej funkcji z serwisu używaj enuma, jeśli enum się zmieni to tutaj TS rzuci błędem